service-binding/postgresql-crunchy/pom.xml

```diff
@@ -39,16 +39,6 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-kubernetes-service-binding</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-junit5</artifactId>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>io.rest-assured</groupId>
-            <artifactId>rest-assured</artifactId>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>io.quarkus.qe</groupId>
             <artifactId>quarkus-test-openshift</artifactId>
```

Comment:
This one is in the parent too, so I think we can remove it from here.

Reply:
I still see `quarkus-test-openshift` here, is it necessary then, or can we remove it?

Reply:
done!